client/src/player/sagas.js

```diff
@@ -43,28 +44,35 @@ function* connectPlayer() {
 }
 
 /**
- * @todo Remove that function once multiplayer feature is available
- * @return {array} of player's ID created
+ * Create one fakePlayer
+ * @param name
+ * @return {{firstname: *, isFakePlayer: boolean, id}}
  */
-export function* createFakePlayers() {
-    const playersId = [];
-    let result;
-    result = yield db.collection('players').add({
-        firstname: 'robot1',
-    });
-    playersId.push(result.id);
+function* createFakePlayer(name) {
+    const fakePlayer = {
+        firstname: name,
+        isFakePlayer: true,
+    };
+    const result = yield db.collection('players').add(fakePlayer);
 
-    result = yield db.collection('players').add({
-        firstname: 'robot2',
-    });
-    playersId.push(result.id);
+    return {
+        ...fakePlayer,
+        id: result.id,
+    };
+}
 
-    result = yield db.collection('players').add({
-        firstname: 'robot3',
-    });
-    playersId.push(result.id);
+/**
+ * @todo Remove that function once multiplayer feature is available
+ * @return {array} of player's
+ */
+export function* createFakePlayers() {
+    const players = [];
```

Comment:
FYI
return new Array(4).map((elem,index) => yield createFakePlayer(`robot${i + 1}`))
(just for the syntaxe JS // Functional Programming -like, but it wouldn't work here due to the `yield`

Reply:
Ok thanks for the tips !